docs/user-guide/running_interaction.md

```diff
@@ -0,0 +1,138 @@
+# Interacting with a Running Simulation
+
+Starting in HELICS 2.4 there is a webserver that can be run with the broker_server  or helics_broker_server.  This requires building with the cmake option `HELICS_ENABLE_WEBSERVER` and using a boost version >=1.70.
+
+## Startup
+The webserver can be started with the option `--http`  For example to run a broker server with zmq and the webserver active for 30 minutes.  The `--duration` is optional and the default is 30 minutes but any time can be specified.  
+
+```sh
+helics_broker_server --http --zmq --duration 30minutes
+```
+
+The HTTP server is configured to only respond on the localhost address port 80.  If you want to configure this it can be done through a configuration file.  The format is json
+
+```json
+{
+    "http":{
+        "port":8080,
+        "interface":"0.0.0.0"
+    }
+}
+```
+then specified on the command line such as 
+
+```sh
+helics_broker_server --http --zmq --config broker_server_config.json
+```
+the configuration will then make the webserver accessible on any interface on port 8080.  
+
+## Making queries
+The running webserver will start a process that can respond to http request. It can accept GET or POST requests as a REST API.   
```

Comment:
```suggestion
The running webserver will start a process that can respond to http requests. It can accept GET or POST requests as a REST API.   
```